app/views/templates/base.php

```diff
@@ -75,6 +75,7 @@
 } ?><?= $title_productname ?></title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Transvision is a Web application targeting the Mozilla localization community, created and maintained by the French Mozilla community">
```

Comment:
One possible suggestion before I forget it: «Tranvision is a tool used by the Mozilla Community to search translations extracted from products and websites. It also provides API access to translation memory, and specific QA features.»

The second sentence might get truncated, but it would still make sense.